keyboards/tmo50/keymaps/pyrol/keymap.c

```diff
@@ -15,37 +15,54 @@
  */
 #include QMK_KEYBOARD_H
 
+#define CALTDEL LCTL(LALT(KC_DEL))
```

Comment:
You could actually just use: 

```suggestion
#define CALTDEL C(A(KC_DEL))
```

Or even: 
```suggestion
#define CALTDEL LCA(KC_DEL)
```